apps/console/src/features/applications/components/settings/sign-on-methods/step-based-flow/authentication-step.tsx

```diff
@@ -137,6 +137,14 @@ export const AuthenticationStep: FunctionComponent<AuthenticationStepPropsInterf
 
     const classes = classNames("authentication-step-container timeline-body", className);
 
+    let isContainsOTPAuthenticator = false;
+
+    step.options.map(option => [
+        IdentityProviderManagementConstants.TOTP_AUTHENTICATOR, 
+        IdentityProviderManagementConstants.EMAIL_OTP_AUTHENTICATOR, 
+        IdentityProviderManagementConstants.SMS_OTP_AUTHENTICATOR ]
+        .includes(option.authenticator) && (isContainsOTPAuthenticator = true));
```

Comment:
I think it is better to move this `(isContainsOTPAuthenticator = true)` into an `if` block.

Reply:
Done